source/gui/settingsDialogs.py

```diff
@@ -1461,6 +1462,21 @@ def makeSettings(self, settingsSizer):
 			characterProcessing.CONFIGURABLE_SPEECH_SYMBOL_LEVELS.index(curLevel)
 		)
 
+		self.symbolLevelWord = settingsSizerHelper.addItem(
+			wx.CheckBox(
+				self,
+				# Translators: The label for a setting in the Speech category
+				label=_("Speak all punctuations and symbols when reviewing by &word"),

```

Comment:
Please document this new option in the user guide.

Reply:
> Please document this new option in the user guide.

And once done, please add to the checkbox a helpId targeting the anchor of the paragraph documenting this new option in order to support a precise context help. Thanks.

Reply:
@lukaszgo1 wrote:
> Please document this new option in the user guide.

@CyrilleB79 wrote:
> please add to the checkbox a helpId targeting the anchor of the paragraph documenting this new option in order to support a precise context help. Thanks.

Godd catch, thanks.
Done as of 7f18e8df8